Sources/VitalHealthKit/HealthKit/Models/CoreModels+Extensions.swift

```diff
@@ -376,6 +376,19 @@ extension HKSampleType {
   }
 }
 
+func quantity(for statistics: HKStatistics, with options: HKStatisticsOptions) -> HKQuantity? {
+  switch options {
+    case .discreteMax:
+      return statistics.maximumQuantity()
+    case .discreteAverage:
+      return statistics.averageQuantity()
+    case .discreteMin:
+      return statistics.minimumQuantity()
+    default:
```

Comment:
Resting heart rate uses the `.mostRecent` default, so you need to handle that here as well